jira.py

```diff
@@ -60,11 +72,15 @@ def download_issue(issue_key: str, fields):
                 print(r)
                 print("Download failed for ticket {}".format(issue_key))
                 if r.status_code == 401:
-                    print("Wrong password")
-                    reset_auth()
+                    _authorization_failed()
                     # go into while True again, ask for password one more time
                     continue
                 if r.status_code == 403:
```

Comment:
in my case update handler on 401 was enough, but may be also need do it (try bearer) also for 403